src/scss/decksView.scss

```
@@ -135,3 +135,186 @@
+
+  .share-deck-buttons {
+    display: flex;
+    justify-content: center;
+    gap: 8px;
+    margin-top: 4px;
+  }
+}
+
+// The public /share/deck/<token> page. Reuses the deck view grid; only the
+// header identity strip and footer are its own.
+//
```

Stylelint (17.14.0)

[error] Unexpected empty comment (scss/comment-no-empty)

(scss/comment-no-empty)